### PR TITLE
User doc update doc for api changes

### PR DESCRIPTION
**Pull request by sujeebant**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,25 +71,23 @@ The User data is synced between the different connected devices only when the Us
...
 
-The function returns an object containing an attribute `link` containing the Url the user has to access to connect to Masq.
+The function returns a Promise resolving to an Url the user has to open to connect to Masq.
 
-Starts the login procedure with Masq and returns a link.
-Once the user access the link and connects to his profile, the login will end, this can be checked with the `masq.logIntoMasqDone()`.
+#### `masq.logIntoMasq(stayConnected) -> Promise<void>`
 
-#### `masq.logIntoMasqDone() -> Promise<void>`
+`stayConnected` is a boolean representing if the User should stay connected until he explicitely signs out or only until the User-app closes.
```
Comment (leventdem) [inline README.md]: Maybe closure instead of closes ?

File: README.md [hunk 2]
```diff
@@ -69,25 +71,23 @@ The User data is synced between the different connected devices only when the Us
...
 
-Returns a promise that resolves when the login procedure ends.
+This function starts the login procedure with an instance of Masq-app
+Once the user accesses the link and connects to his profile, the login will end and the the returned promise will resolve.
```
Comment (leventdem) [inline README.md]: duplication of the the

File: README.md [hunk 3]
```diff
@@ -69,25 +71,23 @@ The User data is synced between the different connected devices only when the Us
...
 Signs out of Masq from Masq.
 This also means the User-app data on the device will stop being synced.
 
-#### `masq.logIntoMasq(stayConnected) -> Promise<ret: object>`
-
-`stayConnected` is a boolean representing if the User should stay connected until he explicitely signs out.
+### `masq.getLoginLink() -> Promise<link: string>`
 
-The function returns an object containing an attribute `link` containing the Url the user has to access to connect to Masq.
+The function returns a Promise resolving to an Url the user has to open to connect to Masq.
```
Comment (leventdem) [inline README.md]: Proposition: The function returns a Promise resolving to an Url, which must be opened by the user in order to connect to Masq.

File: README.md (hunk 1 above)
Comment (sujeebant) [reply README.md]: changed the sentence

File: README.md (hunk 2 above)
Comment (sujeebant) [reply README.md]: :+1:

File: README.md (hunk 3 above)
Comment (sujeebant) [reply README.md]: changed !

**Pull request merged**